examples/encfs/README.md

```diff
@@ -66,22 +75,88 @@ The AAD token with permission to AKV/mHSM can be obtained with the following com
 az account get-access-token --resource https://managedhsm.azure.net
 ```
 
-Replace [AAD token](importkeyconfig.json#L11) in `importkeyconfig.json` with the output accessToken.
-Replace [ENCFS AKV endpoint](encfs-sidecar-args.json#L15) in `enfcs-sidecar-args.json` with the mhsm endpoint.
-
 #### 5. Fill in Key Information
 
-After setting up an [Azure Key Vault resource](#import-key), fill in the `importkeyconfig.json` file with the name of the key to be created and imported into the key vault [Key name](importkeyconfig.json#L3). Additionally, fill in `encfs-sidecar-args.json` with the name of the key to be created and imported into the key vault [Key name](encfs-sidecar-args.json#L9).
+Replace [AAD token](importkeyconfig.json#L11) and [ENCFS AKV endpoint](importkeyconfig.json#L9) in `importkeyconfig.json` with the output accessToken and the mHSM endpoint.
+Replace [ENCFS AKV endpoint](encfs-sidecar-args.json#L15) in `enfcs-sidecar-args.json` with the mHSM endpoint.
+
+After setting up an Azure Key Vault resource, fill in the `importkeyconfig.json` file with the name of the key to be created and imported into the key vault [Key name](importkeyconfig.json#L3). 
+Additionally, fill in `encfs-sidecar-args.json` with the name of the key to be created and imported into the key vault [Key name](encfs-sidecar-args.json#L9).
+
+Additionally, fill in the optional [key derivation](importkeyconfig.json#L14) for RSA keys and [Key type: `RSA-HSM` or `oct-HSM`](importkeyconfig.json#L4) fields or remove these fields from the `importkeyconfig.json` file. 
+Fill in [key derivation](encfs-sidecar-args.json#L18) and [Key type: `RSA-HSM` or `oct-HSM`](encfs-sidecar-args.json#L10) in `encfs-sidecar-args.json` or remove them as well as well. 
+
+The user needs to instantiate an [Azure Storage Container](https://learn.microsoft.com/en-us/azure/storage/common/storage-account-create?toc=%2Fazure%2Fstorage%2Fblobs%2Ftoc.json&bc=%2Fazure%2Fstorage%2Fblobs%2Fbreadcrumb%2Ftoc.json&tabs=azure-portal) onto which the encrypted filesystem will be uploaded.

```

Comment:
Since this isn't part of creating the key, maybe we can put this as a step before this: "5. Create Azure Storage Container"

Reply:
yes, it'll make smaller easier to follow steps

Reply:
ok